static/beta/prod/modules/fed-modules.json

```diff
@@ -15,6 +15,22 @@
             }
         ]
     },
+    "globalLearningResourcesPage": {
+        "manifestLocation": "/apps/learning-resources/fed-mods.json",
+        "defaultDocumentTitle": "All Learning Resources",
+        "modules": [
+            {
+                "id": "globalLearningResourcesPage",
+                "module": "./RootApp",
```

Comment:
@JetyAdam I believe we want this to be `GlobalLearningResourcesPage` per https://github.com/RedHatInsights/learning-resources/blob/master/fec.config.js#L25